pkg/platform/kube/client/deployer.go

```diff
@@ -79,6 +81,18 @@ func (d *Deployer) CreateOrUpdateFunction(ctx context.Context,
 		functionStatus.HTTPPort = functionInstance.Status.HTTPPort
 	}
 
+	if d.platform.GetConfig().SensitiveFields.MaskSensitiveFields {
```

Comment:
allow the user to specify on his function, explicitly, whether it should or should not mask sensitive fields.

Reply:
Do we still want to set it on the platform config then?
Maybe as a default value

Reply:
AND (platformconfig flag, functionconfig flag)